components/org.wso2.carbon.identity.oauth.dcr.endpoint/src/main/resources/api.identity.oauth.dcr.endpoint.yaml

```diff
@@ -73,6 +73,43 @@ paths:
             $ref: '#/definitions/error'
       tags:
         - OAuth2 DCR
+  # Get application info by client name
+    get:
+      summary: |
+        Get OAuth2 application information
+      description: |
+         This API is used to get/retrieve an OAuth2 application by client_name.
+      x-wso2-request: |
+        curl -X GET -H "Authorization: Basic YWRtaW46YWRtaW4=" -H "Content-Type: application/json" -d '{}' "https://localhost:9443/api/identity/oauth2/dcr/v1.0/register?client_name=application_owner_application_1"
```

Comment:
Since we are doing a get here, using 'register' as the context seems misleading. If it's not from a spec, consider changing the context to /application or something similar

Reply:
dcr/v1.0/register is the pattern followed in DCRM spec [1]

[1] https://tools.ietf.org/html/rfc7592#section-2.1

Reply:
Ok, then fine 👍